scripts/zones/Abyssea-Tahrongi/npcs/Ada.lua

```diff
@@ -0,0 +1,19 @@
+-----------------------------------
+-- Zone: Abyssea - Tahrongi
+--  NPC: Ada
```

Comment:
Does the default interaction need the empty script to work? This should also have the pos command to get to this NPC

Reply:
Didn't test it without it, just added it cause I looked through npcs that also had empty scripts also using default interaction so thought it was the norm. I'll go ahead and test and remove if works without it then.

Reply:
tested, working without script.